.gitignore

```diff
@@ -1,63 +1,30 @@
-image/
```

Comment:
We should recover all the files that used to be ignored

Reply:
👍🏻